corehq/apps/app_manager/views/utils.py

```diff
@@ -11,14 +11,32 @@
 from django.utils.translation import ugettext as _
 
 from corehq import toggles
-from corehq.apps.app_manager.dbaccessors import get_app, wrap_app, get_apps_in_domain, get_current_app
+from corehq.apps.app_manager.dbaccessors import (
```

Comment:
Might wanna try out `isort` - not sure how to use that in pycharm, but it's great for cleaning imports, and we have a [HQ config](https://github.com/dimagi/commcare-hq/blob/master/.isort.cfg) for it.